src/pdn/test/power_switch.api.tcl

```diff
@@ -0,0 +1,52 @@
+source "helpers.tcl"
```

Comment:
Add a comment to clarify exactly what this test is checking.

Reply:
Done

Reply:
Done